test/fixed.jl

```diff
@@ -146,6 +146,14 @@ end
     @test floatmin(Q11f4) == Q11f4(0.06)
 end
 
+@testset "isinteger" begin
+    # issue #120
+    for T in (Fixed{Int8,6}, Fixed{Int16,8}, Fixed{Int16,10}, Fixed{Int32,16})
+        a = 1
+        @test all(isinteger.(T.(a)))
```

Comment:
`a` is a scalar here so I don't think you need `all` or the dots.

How about a loop `for a = 0:floor(Int, typemax(T))`? (Here and below.) Or igerf you want to check ones that are non-integers, you could do
```julia
for a = reinterpret(typemin(T)):reinterpret(typemax(T))
    x = reinterpret(T, a)
    @test isinteger(x) == (x == round(Int, x))
end
```
This kind of exhaustive check probably only makes sense for 16bits or less, and probably best put inside a function so it gets compiled.